From 5f8aa012303667918f52100db770324ba4b5831d Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Wed, 15 Nov 2023 00:19:32 +0100
Subject: [PATCH 1/2] Fix unknown cmake command in error paths

CMake Error at CMakeLists.txt:88 (msg):
  Unknown CMake command "msg".
---
 CMakeLists.txt              | 2 +-
 cmake/modules/QmlTest.cmake | 2 +-
 test/qml/CMakeLists.txt     | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e14b1b5..e70f8ec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -85,7 +85,7 @@ endif()
 find_program(qmltestrunner_exe qmltestrunner)
 
 if(NOT qmltestrunner_exe)
-  msg(FATAL_ERROR "Could not locate qmltestrunner.")
+  message(FATAL_ERROR "Could not locate qmltestrunner.")
 endif()
 
 macro(add_qml_test COMPONENT_NAME)
diff --git a/cmake/modules/QmlTest.cmake b/cmake/modules/QmlTest.cmake
index 9c0bbf0..87fab8e 100644
--- a/cmake/modules/QmlTest.cmake
+++ b/cmake/modules/QmlTest.cmake
@@ -18,7 +18,7 @@
 find_program(qmltestrunner_exe qmltestrunner)
 
 if(NOT qmltestrunner_exe)
-  msg(FATAL_ERROR "Could not locate qmltestrunner.")
+  message(FATAL_ERROR "Could not locate qmltestrunner.")
 endif()
 
 macro(add_qml_test SUBPATH COMPONENT_NAME)
diff --git a/test/qml/CMakeLists.txt b/test/qml/CMakeLists.txt
index 38e7253..e817ea0 100644
--- a/test/qml/CMakeLists.txt
+++ b/test/qml/CMakeLists.txt
@@ -4,7 +4,7 @@ add_custom_target(qmlunittests_1_1)
 find_program(qmltestrunner_exe qmltestrunner)
 
 if(NOT qmltestrunner_exe)
-  msg(FATAL_ERROR "Could not locate qmltestrunner.")
+  message(FATAL_ERROR "Could not locate qmltestrunner.")
 endif()
 
 set(qmltest_command
-- 
2.42.1

